lib/osf-components/addon/components/institutions-widget/component.ts

```diff
@@ -18,18 +18,20 @@ import template from './template';
 @layout(template, styles)
 export default class InstitutionWidget extends Component.extend({
     loadNodeAffiliatedInstitutions: task(function *(this: InstitutionWidget) {
-        if (!this.node) {
+        if (!this.node || !this.shouldShowModal) {
             return undefined;
         }
 
         try {
-            const affiliatedList: QueryHasManyResult<Institution> =
-                yield this.node.queryHasMany('affiliatedInstitutions');
-            this.set('affiliatedList', affiliatedList);
+            const affiliatedList: QueryHasManyResult<Institution> = yield this.node.sparseLoadAll(
```

Comment:
```suggestion
            const affiliatedList: SparseModel[] = yield this.node.sparseLoadAll(
```


^ import from `ember-osf-web/utils/sparse-fieldsets`